.travis.yml

```diff
@@ -1,32 +1,29 @@
 sudo: required
 language: go
-
 go:
 - 1.13.x
-
 script:
 - make dist
-
 before_deploy:
 - "./ci/hashgen.sh"
-
 deploy:
   provider: releases
   api_key:
-    secure: aYgBmrFjqAzOBWncNQ2EKKFZJwOD2+l+a7Lll8+DVJkNyvLdzliI8iD4ro5KLEIwJjlghwM/Xy78O5oMlTPGoBf6IwbFE4nxKr0wlwFonSym5XCW2W7diGwxNvJUvBqY3wHjV13UpFNJVOqKbcaAVC+oGP6Wt4vNttroUaikpokwTfDXneksg5YIIemKkNXAvvWyRHrdOUgn8U8qLnaJ5gdpdcQOACsVXoI43ogsX49nSqXVIaBqt1Ndz+53Dsu7nSUwuwg0CnTYT05sU83RUhzRckvoE/l3mmd/FQBmVyDTfTwjrVwQLaKv2ldqlDmasV1qQ32bsEAfnGswbEZcJ0Qpx6VasAtDk69OW0RPyHs/VMw9BHVgoQyDaO2KGTKBJBBsewaIcJPBnWcDoGsv8oV/s/6tagqT+20zobGLqJfu3JxPlG7m+mC17NW8Fe9UslexyP9R1PgJfghGqQisDIrid9KixqfMpfTAqLykVGR3ra5NctKJEvpM2lyjLsxKxgzRAeIcd/i6Rt/eQX7lQAFQuP+Gq/hwxffFFUvUJPaBg5xk69S5QF6OrVof/LJjReE39u5z+q3O4f1auyZXSlE3ZNTWJEZ2DCuD5fLQm1b2O8sTAuK2HXnPr2gLvjUTO5QaRlQiVwxeYNcaMol42VFZ+XpBU+PQOuhSUmpqhOk=
+    secure: P+idRN1yOgZuXWslhPJ0HHaaOZJh+MHvWdfSMbFP8wOL7RCHpoMFe+Trpf0oic83ZD3bGOsEhOeBr2JOsjbJi31gBTwKSNTHKCiz1N8MMNiJ3q7PJHuV8X40j4S0XxHbczFyKCzpL1R1HTA8XN2BWHKYg+8itiRSTLzJeZO7D4Xkd8iaRg3ArK3MtHlWHVXowDJV03daDcZFmxj40jOKvdUbwB941Lp7B9OupwQL6e1tO+4pi5zt9JqaO4G9VAfElm7xoREzGgLXN2ijq8URdLwD6IRfkuUGEe40rCXqMAtIQdnmf61EzrMVrVyjtGUEgBd5TcwNKl2FPzCSVAk2ySxmGkskV0vxJwhNrIqYnpfBFaPy4r9o4+plLgfhvKTGbuaqVgHDykNbiwXW5nZtmTvFJ+/aMSn0h0cz7Uu7Pj2XwE7+vOjQSUjms/oTGFI+S0+OlfVEvlnjgkUbDO6u3vxFz0vaf6+7P+flkhqPILgGr84Kgbqa26XFtDKy9LGNdKV4O5h7zOrYvHu0WOX5MmZ4NHbBQU9mLYkeskoss86m/Ga227x86M4cIZVWk9dVJrMnrurIW1A/G9XO4ylmulrxPFfR4GCBnrz9Lf8nX96eUlRXxemU/giO57HLXetutqqPKAQQHy4vGDllQVHj/mvbtXudDZ6QjkDehWfyD30=
```

Comment:
Once ready, and signed-off, we will need to revert the secure token to the original value, since the value in the PR is for testing on your account, we will leave it until then.